src/Tokenizer/TokenInterface.php

```diff
@@ -7,6 +7,7 @@ interface TokenInterface
     const TYPE_WHITESPACE = "WS";
     const TYPE_COMMENT_MULTILINE = "COMMENT_MULTILINE";
     const TYPE_COMMENT_ONELINE = "COMMENT";
+    const TYPE_COMMENT_LINE = "COMMENT_LINE";
```

Comment:
I'm unsure what the `COMMENT_LINE` token is supposed to represent. IIRC, the entire block comment should already be represented by the `COMMENT_MULTILINE` token, shouldn't it?